pkg/collector/python/loader.go

```diff
@@ -257,11 +257,16 @@ func reportPy3Warnings(checkName string, checkFilePath string) {
 
 		if warnings, err := validatePython3(checkName, checkFilePath); err != nil {
 			status = a7TagUnknown
+			log.Errorf("Failed to validate Python 3 linting for check '%s': '%s'", checkName, err)
 		} else if len(warnings) == 0 {
 			status = a7TagReady
 			metricValue = 1.0
 		} else {
 			status = a7TagNotReady
+			log.Warnf("The Python 3 linter returned the following warnings for check '%s':", checkName)
+			for _, warning := range warnings {
+				log.Warn(warning)
```

Comment:
Is `Warn` the right level? I've heard from @therve that the linter could find tens to hundreds of issues for a given check, if that's the case I think we should warn them with `Debug` for example, and mention in the previous log message to `set the log level to "debug" and restart the Agent to have the linter warnings logged` for example